src/components/Disclosure.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <Disclosure>
+    <DisclosureButton
+      class="flex w-full flex-row justify-between px-0 my-4 focus:ring-1 ring-offset-8 rounded-md type-subtitle-2"
+      style="text-align:left"
```

Comment:
There's a tailwind class for this, https://tailwindcss.com/docs/text-align#setting-the-text-alignment